### PR TITLE
Under issue #3, feature added.

### PR DESCRIPTION
**Pull request by hmangukia**
Feature of getting the latest 10 tweets from Twitter is added. Just say "get tweets" and latest 10 tweets will be shown with the content of the tweet, username of the creator and the time when it got created.

### CODE REVIEW

File: windows/benji.py [hunk 1]
```diff
@@ -80,8 +83,18 @@ def events(frame,put):
 		cv2.imwrite(path + "/" + str(name) + ".jpg", img)
 		cam.release()
 		cv2.destroyAllWindows()
-
-    #Screenshot    
+	
+	#get top 10 tweets
+	elif link[0] =="get" and link[-1] == "tweets":
```
Comment (sara-02) [inline windows/benji.py]: You missed a space after `== "get"`

File: windows/benji.py [hunk 2]
```diff
@@ -80,8 +83,18 @@ def events(frame,put):
...
-    #Screenshot    
+	
+	#get top 10 tweets
+	elif link[0] =="get" and link[-1] == "tweets":
+		auth = OAuthHandler(twitterCredentials.consumer_key, twitterCredentials.consumer_secret)
+		auth.set_access_token(twitterCredentials.access_token, twitterCredentials.access_secret)
+		api = tweepy.API(auth)
+
+		for status in tweepy.Cursor(api.home_timeline).items(10):
+			print("\n",status.text)
```
Comment (sara-02) [inline windows/benji.py]:
```python
    print("\n", status.text)
    print("By ", status.user.screen_name, " at ", status.user.created_at)
```
Missing the spacing after `,` in the print statement.

**Comment by hmangukia** [comment]
Hey @sara-02 I have made the changes you suggested :) Do I need to make a pull request again?

**Comment by sara-02** [comment]
@hmangukia No need to create a new pull request. We are good with these changes. @the-ethan-hunt please add your reviews as well.

**Comment by the-ethan-hunt** [comment]
Great review @sara-02 . No need to review it again :smile:

**Pull request merged**